tests/acceptance/features/bootstrap/ArchiverContext.php

```diff
@@ -212,25 +211,49 @@ public function userDownloadsTheArchiveOfTheseItems(
 	 *
 	 * @param string $type
 	 * @param TableNode $expectedFiles
+	 * @param string $responseData
 	 *
 	 * @return void
 	 *
 	 * @throws Exception
 	 */
-	public function theDownloadedArchiveShouldContainTheseFiles(string $type, TableNode $expectedFiles):void {
+	public function theDownloadedArchiveShouldContainTheseFiles(string $type, TableNode $expectedFiles, string $responseData=""):void {
 		$this->featureContext->verifyTableNodeColumns($expectedFiles, ['name', 'content']);
+		$contents = $this->featureContext->getResponse()->getBody()->getContents();
 		$tempFile = \tempnam(\sys_get_temp_dir(), 'OcAcceptanceTests_');
 		\unlink($tempFile); // we only need the name
 		$tempFile = $tempFile . '.' . $type; // it needs the extension
-		\file_put_contents($tempFile, $this->featureContext->getResponse()->getBody()->getContents());
-		$archive = UnifiedArchive::open($tempFile);
```

Comment:
UnifiedArchive package could not open the tar file while downloading space
`Cannot access phar file entry '/' in archive` error is thrown

Reply:
downloaded space archive has `.` folder in it and `UnifiedArchive::open()` seems to have trouble opening it. And throw the above error.

So, we implemented this new way to check folder/file existence as well as the content of the file